meteor/server/api/blueprints/context/adlibActions.ts

```diff
@@ -431,6 +387,57 @@ export class ActionExecutionContext extends ShowStyleContext implements IActionE
 
 		return clone(unprotectObject(newPartInstance))
 	}
+	updatePartInstance(part: 'current' | 'next', props: Partial<IBlueprintMutatablePart>): IBlueprintPartInstance {
```

Comment:
me & @Julusian talked about this one the other day, I noted down that initially we won't allow editing the current part due to having to handle side-effects, but we'll keep `part: 'next'` for future compatibility.

Reply:
I did that for removePieceInstances, but not for updatePartInstance. 
I think that keeping this one to edit the current is reasonable, and matches the flexibility given with updatePieceInstance, and is generally safer/less destructive than removePieceInstances

Reply:
never mind me, I got things mixed up here as well..